specification/sql/resource-manager/readme.md

```diff
@@ -49,7 +49,6 @@ input-file:
 - Microsoft.Sql/2014-04-01/geoBackupPolicies.json
 - Microsoft.Sql/2014-04-01/importExport.json
 - Microsoft.Sql/2014-04-01/metrics.json
-- Microsoft.Sql/2014-04-01/operations.json

```

Comment:
Why are we removing this reference without removing the files?

Reply:
2014-04-01 operations is still supported, so it is still in swagger. However client code gen should use 2015-05-01-preview now.